app/models/journal_entry.rb

```diff
@@ -1,4 +1,5 @@
 class JournalEntry < ApplicationRecord
   validates :title, presence: true
   validates :content, presence: true
+  validates :user_id, presence: true
```

Comment:
Why not a belongs_to association ? We don't add ids like this 🤔 
[2.1 The belongs_to Association](https://guides.rubyonrails.org/association_basics.html#the-belongs-to-association)

Reply:
ok thank you i changed it to a belongs to association